shared/locales/en/website-common.json

```diff
@@ -7,7 +7,7 @@
 		"faq": "FAQ",
 		"finances": "Finances",
 		"how-it-works": "How It Works",
-		"my-profile": "My Profile",
```

Comment:
The link will say "Login" now if the user is logged in, which is wrong in my opinion. We should make it dynamic based on the fact if the user is logged in. But as long that's not the case, I would say that "My Profile" works better.

Reply:
if user sees this the same text string when logged in, then "login" would be definitely confusing. changed it back to My Profile. We can discuss at a later stage to add different terms depending on login/logout state.